bin/common.bash

```diff
@@ -568,7 +568,7 @@ with_kubeconfig() {
     else
         log_info "Using unencrypted kubeconfig ${kubeconfig}"
         # shellcheck disable=SC2048
-        KUBECONFIG=${kubeconfig} ${*}
+        KUBECONFIG=${kubeconfig} "$@"
```

Comment:
I feel a bit troubled that this has evaded our eyes for so long, good that it gets fixed. :+1:

Reply:
Isn't that what that [shellcheck directive](https://www.shellcheck.net/wiki/SC2048) disables?